lib/email/sender.rb

```diff
@@ -442,9 +444,11 @@ def fix_parts_after_attachments!
 
         html_part = @message.html_part
         @message.html_part = nil
+        @message.parts.reject! { |p| p.content_type.start_with?("text/html") }
```

Comment:
Setting `html_part` to `nil` is required for [this test](https://github.com/discourse/discourse/blob/main/spec/lib/email/sender_spec.rb#L621) to pass. Rejecting parts is required for the activity summary one.

I tried to figure out why, but there's something arcane going on here in the mail gem, so I eventually had to give up and accept that it works.